changelog.md

```diff
@@ -1,4 +1,5 @@
 # vNEXT
+- [core] Ambiguity of where to set environment variable is fixed. now you can set environment related with the control panel at the root directory of `controlpanel` and environemt variables related with the engine under root directory of `minimal`.
```

Comment:
can you add a new section about breaking changes, because it looks like various env's like LANG, CURRENCY and COUNTRY cannot be used anymore and the prefixed env variables have to be used instead, that is breaking.